receiver/webhookeventreceiver/metadata.yaml

```diff
@@ -3,6 +3,6 @@ type: webhookevent
 status:
   class: receiver
   stability:
-    development: [logs]
-  distributions:  
+    alpha: [logs]
+  distributions: [contrib] 
```

Comment:
To add contrib here, it needs to be added in the https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/manifest.yaml first

Reply:
gotcha, removing contrib!